README.md

```diff
@@ -265,6 +265,11 @@ There are following known limitations of native OTel logs collection:
   Logs collection with fluentd is still recommended if chart deployed with `autodetect.istio=true`.
 - Journald logs cannot be collected natively by Splunk OTel Collector yet.
 
+### Running it as non root user
```

Comment:
Suggested change:

"Running the container in non-root user mode"

"Collecting logs often requires reading log files that are owned by the `root` user. By default, the container runs with `securityContext.runAsUser = 0`, which gives the `root` user permission to read those files. To run the container in `non-root` user mode, set `.Values.otelAgent.runInitForNonRoot` to `true` to cause the container to run the required file system operations as UID and GID `20000`.

**Note:** `cri-o` container runtime did not work during internal testing."